composer.json

```diff
@@ -14,5 +14,10 @@
     "psr-4": {
       "BeechIt\\SolrfalTextextract\\": "Classes/"
     }
+  },
+  "extra": {
+    "typo3/cms": {
+      "extension-key": "solrfal-textextract"
```

Comment:
This should be the extension key. so with a `_` instead of `-`

Reply:
d55e83d530edcce602d29c6d26e53d95bbd9afe0